lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -531,7 +532,7 @@ public function getAlterTableSQL(TableDiff $diff)
         }
 
         foreach ($diff->changedColumns as $columnDiff) {
-            /** @var $columnDiff \Doctrine\DBAL\Schema\ColumnDiff */
+            assert($columnDiff instanceof ColumnDiff);
```

Comment:
This assertion, as well as the original doc block, may be redundant.

Reply:
Even psalm says so (as an info level)